src/models/propulsion/FGBldc.cpp

```diff
@@ -0,0 +1,224 @@
+/*%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+ Module:       FGBldc.cpp
+ Original Author:         Matt Vacanti
+ Corrections and updates: Paolo Becchi
+ Date started: 11/08/2020
+ Corrected   : 26/12/2021
+ Purpose:      This module models an DLDC electric motor
+
+ --------- Copyright (C) 2020  Matt Vacanti / Paolo Becchi-------------
+
+UPDATE LICENSE
+
+FUNCTIONAL DESCRIPTION
+--------------------------------------------------------------------------------
+
+UPDATE FUNCTIONAL DESCRIPTION
+
+HISTORY
+--------------------------------------------------------------------------------
+11/08/2020  MDV  Created
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+INCLUDES
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
+
+#include <iostream>
+#include <sstream>
+#include <math.h>
+
+#include "FGFDMExec.h"
+#include "FGBldc.h"
+#include "FGPropeller.h"
+#include "input_output/FGXMLElement.h"
+
+using namespace std;
+
+namespace JSBSim {
+
+/*%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+CLASS IMPLEMENTATION
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
+
+FGBldc::FGBldc(FGFDMExec* exec, Element *el, int engine_number, struct FGEngine::Inputs& input)
+  : FGEngine(engine_number, input)
+{
+      Load(exec,el);
+
+      Type = etElectric;
+      PowerWatts = 745.7;
+      hptowatts = 745.7;
+
+    // this property is not necessary since is computed using other properties
+    // if (el->FindElement("maxcurrent"))
+    // MaxCurrent= el->FindElementValueAsNumber("maxcurrent");
+
+      if (el->FindElement("maxvolts"))
+        MaxVolts= el->FindElementValueAsNumber("maxvolts");
+
+      if (el->FindElement("velocityconstant"))
+        VelocityConstant= el->FindElementValueAsNumber("velocityconstant");
```

Comment:
```suggestion
      if (el->FindElement("velocityconstant"))
        VelocityConstant= el->FindElementValueAsNumberConvertTo("velocityconstant", ***);
```
Given that `velocityconstant` seems to be of the same dimension than a current, is `velocityconstant` a proper name for that parameter ?
Please request a unit by using `FindElementValueAsNumberConvertTo` instead of `FindElementValueAsNumber`.